includes/modules/payment/paypal/paypalwpp_incontext.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * paypal EC In-Context button display template
+ *
+ * @package paymentMethod
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: Author: mc12345678  Wed Aug 17 03:03:03 2016 -0500 Modified in v1.6.0 $
+ */
+
+    require_once(DIR_WS_CLASSES . 'payment.php');
+    $paypalwpp_module = 'paypalwpp';
+    // init the payment object
+    $classPay = new payment($paypalwpp_module);
+?><script>
+window.paypalCheckoutReady = function () {
+  paypal.checkout.setup('<?php echo MODULE_PAYMENT_PAYPALWPP_MERCHANTID; ?>', {
+    //locale: '<?php echo ${$paypalwpp_module}->getLanguageCode('incontext'); ?>',
```

Comment:
Found the code snippet from which this originated...

ipn_main_hander.php:

```php
  // See if the paypalwpp module is enabled.
  if (defined('MODULE_PAYMENT_PAYPALWPP_STATUS') && MODULE_PAYMENT_PAYPALWPP_STATUS == 'True') {
    $paypalwpp_module = 'paypalwpp';
    // init the payment object
    $payment_modules = new payment($paypalwpp_module);
    // set the payment, if they're hitting us here then we know
    // the payment method selected right now.
    $_SESSION['payment'] = $paypalwpp_module;
    // check to see if we have a token sent back from PayPal.
    if (!isset($_SESSION['paypal_ec_token']) || empty($_SESSION['paypal_ec_token'])) {
      // We have not gone to PayPal's website yet in order to grab
      // a token at this time.  This will send the customer over to PayPal's
      // website to login and return a token
      $$paypalwpp_module->ec_step1();
    } else {
      // This will push on the second step of the paypal ec payment
      // module, as we already have a PayPal express checkout token
      // at this point.
      $$paypalwpp_module->ec_step2();
    }
  }
```

While the above code is a "snippet" from the entire file, $payment_modules is not used at all in the remainder of that code other than to be assigned or potentially assigned again elsewhere later in the same file.  Further, the use of $$paypalwpp_module instead of ${$paypalwpp_module} while apparently clear enough for PHP 7.0 and doesn't currently cause an issue, seems like perhaps could lead to a problem in the future if not considered, but not trying to make changes just to do so. (Like the missing semi-colon in the tpl_ec_button file in the non-replaced link line.)